frontend/src/scenes/urls.ts

```diff
@@ -45,4 +41,11 @@ export const urls = {
     personalization: () => '/personalization',
     ingestion: () => '/ingestion',
     onboardingSetup: () => '/setup',
+    // Cloud only
+    organizationBilling: () => '/organization/billing',
+    billingSubscribed: () => '/billing/subscribed',
```

Comment:
Nit: should we keep the same URL structure? So it would be 'organization/billing/subscribed'

Reply:
Makes sense!